infra/tempo/config/tempo.yaml

```
@@ -0,0 +1,62 @@
+metrics_generator_enabled: true
+search_enabled: true
```

```suggestion
search_enabled: true

multitenancy_enabled: false
```

This is a replacement for `auth.enabled=false` in `docker-compose.yaml`. The latter is deprecated.